v2/media.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022, Google Inc.
```

Comment:
I think this could just be content_type.go given that it doesn't have other media functionality

Reply:
Good call.